tests/unit_tests/data_steward/utils/pipeline_logging_test.py

```diff
@@ -60,16 +62,30 @@ def assert_logs_handled(self, current_level, mock_file_emit,
                   for (log_record,), _ in mock_stream_emit.call_args_list]
         self.assertListEqual(expected, actual)
 
-    @mock.patch('logging.FileHandler._open')
-    def test_configure(self, mock_open):
+    def assert_correct_log_filename(self, file_handler: logging.FileHandler,
+                                    expected_basename: str):
         """
-        Verify that root level and handlers are properly set after configure
-        :param mock_open: mock to prevent side effect of opening file
+        Verifies that the provided logging.FileHandler instance would be writing to a file whose name matches
+        expected_basename
+        :param file_handler: logging.FileHandler instance to test
+        :param expected_basename: expected basename of log file
+        """
+        log_filename = file_handler.baseFilename
+        log_basename = os.path.basename(log_filename)
+        # use endswith as opposed to .assertEquals so we don't care about slash type.
+        self.assertTrue(log_filename.endswith(expected_basename))
```

Comment:
The variable `log_basename` is unused. Maybe you intended to do this (below) but then decided against it?
```suggestion
        self.assertEquals(log_filename, expected_basename)
```
It seems to me that it'd better account for a scenario like this, no?
```
log_filename='abc20210409-project.log'
expected_basename ='20210409-project.log'
```

Reply:
You are correct, it was an abandoned attempt at doing a direct filename match.  It turns out python's basename methods don't properly handle paths in some cases: https://stackoverflow.com/questions/678236/how-to-get-the-filename-without-the-extension-from-a-path-in-python.  I ran into this in my own testing, locally.

To keep this code simple and to not have to deal with path markup differences, I chose to simply compare using `.endswith()`.

Reply:
I have updated the test to be slightly more intelligent about its assertions.